snippets/routing-odc.adoc

```diff
@@ -0,0 +1,16 @@
+// Text snippet included in the following modules:
+//
+// * modules/odc-importing-codebase-from-git-to-create-application.adoc
+
+:_content-type: SNIPPET
+
+Routing:: Click the *Routing* link to:
```

Comment:
I’d suggest that the lead-in sentence not be a fragment (per ISG). So something like, “By clicking the *Routing* link, you can perform the following actions:”